README.md

```diff
@@ -22,7 +22,19 @@ On Windows 10 you can either build for win32 or use XMing and run your builds in
 
 For more information about how to build on the various systems, refer to the platform specific docs in the `docs` folder!
 
-# Overview
+# More docs!
+
+Refer to the OS specific docs:
+
+* [Linux users](docs/Linux.md)
+* [Windows users](docs/Windows.md) or if you are [Visual Studio user](docs/VisualStudio.md)
```

Comment:
Should mention that the first option is WSL/MinGW. Also if you replace 'OS specific' with 'platform specific', then you can list Emscripten and 32blit with the others.

Reply:
I wonder if- faced with good options for using Visual Studio- anyone is actually likely to use WSL/MinGW or WSL/XMing (the latter also doesn't support audio with the current mainline WSL)? We should maybe also move that documentation into a section specifically for Windows uers.